src/integrationTest/java/uk/gov/hmcts/reform/professionalapi/util/ProfessionalReferenceDataClient.java

```diff
@@ -299,6 +300,24 @@ private Map getResponse(ResponseEntity<Map> responseEntity) {
         }
 
 
+        return getResponse(responseEntity);
+    }
+
+    public  Map<String, Object> editPaymentsAccountsByOrgId(PbaEditRequest pbaEditRequest, String orgId, String hmctsAdmin) {
+        ResponseEntity<Map> responseEntity = null;
+        String urlPath = "http://localhost:" + prdApiPort + APP_INT_BASE_PATH + "/" + orgId + "/pbas";
+
+        try {
+            HttpEntity<PbaEditRequest> requestEntity = new HttpEntity<>(pbaEditRequest, getMultipleAuthHeaders(hmctsAdmin));
+            responseEntity = restTemplate.exchange(urlPath, HttpMethod.PUT, requestEntity, Map.class);
+        } catch (RestClientResponseException ex) {
+            HashMap<String, Object> statusAndBody = new HashMap<>(2);
```

Comment:
2 not required the hashmap will dynamically resize

Reply:
I did not know this, thanks for the comment. I will make this change.